tests/scripts/da/autotune_layer_config_id_tests.sh

```diff
@@ -327,6 +302,14 @@ function validate_autotune_tunables_api() {
 			if [ "${config_id_test_name}" == "${autotune_layer_config_id_tests[3]}" ]; then
 				kubectl delete -f ${new_config_yaml} -n monitoring >> ${LOG}
 			fi
+
+			echo "Deleting autotune layer config yaml ${test_layer_config}..." | tee -a ${LOG}
+			kubectl delete -f ${config_yaml} -n monitoring | tee -a ${LOG}
```

Comment:
Ideally the `monitoring` ns needs to be set as a config variable. It should be set based on the cluster type (Eg. minikube, openshift etc)

Reply:
Thanks, updated it.